.github/workflows/python-test-coverage.yml

```diff
@@ -5,11 +5,14 @@ on:
     branches: ["main", "feature*"]
     paths:
       - "python/**"
+env:
+  # Configure a constant location for the uv cache
+  UV_CACHE_DIR: /tmp/.uv-cache
 
 jobs:
   python-tests-coverage:
     runs-on: ubuntu-latest
-    continue-on-error: true
+    continue-on-error: false
```

Comment:
I've noticed over a few past PR iterations this bool flips from true -> false -> true -> false?